tests/recipes/test_gevent.py

```diff
@@ -1,21 +1,11 @@
 import unittest
 from mock import patch
-from pythonforandroid.archs import ArchARMv7_a
-from pythonforandroid.build import Context
-from pythonforandroid.recipe import Recipe
+from tests.recipes.recipe_ctx import RecipeCtx
 
 
-class TestGeventRecipe(unittest.TestCase):
+class TestGeventRecipe(RecipeCtx, unittest.TestCase):
```

Comment:
Also here, I think we can kill the `TestCase` inheritance, can't we?

Reply:
The same than [this](https://github.com/kivy/python-for-android/pull/1982/files#r324457377)